internal/cluster/distribution/pke/pkeaws/pkeawsworkflow/workflow_update_node_pool.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkeawsworkflow
+
+import "github.com/banzaicloud/pipeline/internal/cluster/distribution/pke"
+
+const UpdateNodePoolWorkflowName = "pkeaws-update-node-pool"
+
+type UpdateNodePoolWorkflowInput struct {
+	ProviderSecretID string
+	Region           string
+
+	StackName string
+
+	OrganizationID  uint
+	ClusterID       uint
+	ClusterSecretID string
+	ClusterName     string
+	NodePoolName    string
+
+	NodeImage string
```

Comment:
**Note**: I believe we are going to need a node pool version here as well.

Also it would be great to implement the update with volume sizes as well.

(As long as we want to use the same input in enterprise.)

Reply:
I added the version parameter in a subsequent commit.